docs/source/gbd2017_models/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -0,0 +1,353 @@
+     - BEP vs. nothing on WLZ score at 6 months
+     - none
+     - none
+     - +0.3 (±0.1)
+     - TPP target
+   * - *c_laz*
+     - Spearman correlation co-efficient for birthweight and LAZ
+     - :download:`spreadsheet <child_growth_birthweight_correlations.xlsx>`
+     - MAL-ED study
+     - same
+     - same  
+   * - *c_wlz*
+     - Spearman correlation co-efficient for birthweight and LAZ
+     - :download:`spreadsheet <child_growth_birthweight_correlations.xlsx>`
+     - MAL-ED study
+     - same
+     - same  
+
+
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+
+• IFA: Iron and folic acid supplementation 
+- Dosage: Combined pill or separate pill (30-60 mg iron, 400 μg folic acid)
+- Duration: as soon as possible during pregnancy 
+- Infant outcomes affected: increases birthweight(g)
+
+• MMN: multi-micro nutrient supplementation
+- Multiple micronutrients defined as supplementation with at least 5 micronutrients including the UNIMMAP formulation: 2 mg copper, 65 μg selenium, 800 μg RE vitamin A, 1.4 mg vitamin B1, 1.4 mg vitamin B2, 18 mg niacin, 1.9 mg vitamin B6, 2.6 μg vitamin B12, 70 mg vitamin C, 5 μg vitamin D, 10 mg vitamin E and 150 μg iodine, 30 mg iron, 400 μg folic acid, 15 mg zinc
+- Infant outcomes affected (MMN): reduces preterm births (<37 weeks)-CIs slightly spans 1, reduces low birth weight, increases birthweight(g), reduces small-for gestational age 
+
+• BEP:  balanced energy protein supplementation
+- Dosage: supplements in which protein provides less than 25% of the total energy content
+- Duration: administered 6 months during pregnancy and 6 months during lactation
+- Infant outcomes affected:  increases birthweight(g), reduces small for gestational age
+
+*The intervention ‘MMN’ refers to a single pill that includes iron and folic acid and intervention ‘BEP’ refers to a nutrient bar that includes micronutrients as well as iron and folic
+
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+
+• Baseline: Offering IFA to any women attending ANC (at empirical baseline coverage of IFA)
+• Scenario 1: Offering MMN to any women attending ANC (intervention at target 90% coverage at ANC)
+• Scenario 2: Offering BEP (current evidence effect size) to any women attending ANC (intervention at target 90% coverage at ANC)
+• Scenario 3: Offering BEP (current evidence effect size) to targeted undernourished women, and MMN to any other women attending ANC (intervention at target 90% coverage at ANC)
+• Scenario 4: Offering BEP (hope and dreams effect size) to any women attending ANC (intervention at target 90% coverage at ANC)
+• Scenario 5: Offering BEP (hope and dreams effect size) to targeted undernourished women, and MMN to any other women attending ANC (intervention at target 90% coverage at ANC)
+
+.. _5.0:
+
+5.0 Vivarium modelling components
++++++++++++++++++++++++++++++++++
+
+.. _5.1:
+
+5.1 Vivarium concept model 
+--------------------------
+
+  .. image:: vivarium_conceptdiagram_bep.svg
```

Comment:
This isn't rendering correctly for me either :/

Reply:
aww man...it is rendering correctly in my make html though...

Reply:
Hmm, weird! Maybe it's just me then.